docs/UserGuide.md

```diff
@@ -268,9 +268,9 @@ Format: `editLoan INDEX VALUE REASON`
 
 Examples:
 
-* `editLoan 2 30 bought logistics`
-* `editLoan 3 -30 return money from logisitics`
-* `list` followed by `editLoan 1 -20 return money` will edit the 1st person in the SectresBook,
+* `editLoan 2 amt/30 reason/bought logistics`
+* `editLoan 3 amt/-30 reason/return money from logisitics`
```

Comment:
Wait sorry, recently i inserted the ability to edit by name with the same method, can you also add
`editLoan alex amt/-30 reason/return money from logistics` to indicate either edit by INDEX <OR> NAME

Reply:
Resolved please check!